src/server/routes/api/user/urls/index.ts

```diff
@@ -62,7 +62,15 @@ export default fastifyPlugin(
           );
 
         let maxViews: number | undefined;
-        const returnDomain = req.headers['x-zipline-domain'];
+
+        let returnDomain;
+        const headerDomain = req.headers['x-zipline-domain'];
+        if (headerDomain) {
+          const domainArray = headerDomain.split(',');
+          returnDomain = domainArray[Math.floor(Math.random() * domainArray.length)].trim();
+        } else {
+          returnDomain = req.headers['x-zipline-domain'];
+        }
```

Comment:
don't need this either since the return domain should just be undefined if there is no `headerDomain`